generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -3382,3 +3384,14 @@ def write_to_teds_from_file(
             grpc_types.WriteToTEDSFromFileRequest(
                 physical_channel=physical_channel, file_path=file_path,
                 basic_teds_options_raw=basic_teds_options))
+
+def _assign_numpy_array(numpy_array, grpc_array):
+    """ Assigns the grpc array to the numpy array, while still maintaining the 
+    original shape of the numpy array. 
+    
+    The numpy array assignment is skipped when the length of the grpc array 
+    is greater than the numpy array as this might result in data loss.
```

Comment:
Rerun the codegen script.

https://github.com/ni/nidaqmx-python/actions/runs/4936084221/jobs/8823191714?pr=330
```
Run git diff --exit-code
diff --git a/generated/nidaqmx/_grpc_interpreter.py b/generated/nidaqmx/_grpc_interpreter.py
index 5b9d798..d8253b2 [1](https://github.com/ni/nidaqmx-python/actions/runs/4936084221/jobs/8823191714?pr=330#step:9:1)00644
--- a/generated/nidaqmx/_grpc_interpreter.py
+++ b/generated/nidaqmx/_grpc_interpreter.py
@@ -3388,9 +3388,6 @@ class GrpcStubInterpreter(BaseInterpreter):
 def _assign_numpy_array(numpy_array, grpc_array):
     """ Assigns the grpc array to the numpy array, while still maintaining the 
     original shape of the numpy array. 
-    
-    The numpy array assignment is skipped when the length of the grpc array 
-    is greater than the numpy array as this might result in data loss.
     """
     grpc_array_size = len(grpc_array)
     assert numpy_array.size >= grpc_array_size
Error: Process completed with exit code 1.
```

Reply:
regenerated the files properly